ui/src/literature/containers/LiteratureSearchContainer.jsx

```diff
@@ -248,15 +248,16 @@ function LiteratureSearch({
                     </Row>
                   )}
                 />
-                {getConfigFor('SEARCH_FEEDBACK_CARD_FEATURE_FLAG') && (
-                  <EventTracker
-                    eventCategory="Feedback modal"
-                    eventAction="Open"
-                    eventId="Bottom of results"
-                  >
-                    <SearchFeedback style={{ marginBottom: '10px' }} />
-                  </EventTracker>
-                )}
+                {getConfigFor('SEARCH_FEEDBACK_CARD_FEATURE_FLAG') &&
+                  !embedded && (
```

Comment:
Nice, thank you! can we move this to a variable on top, sth like `const showFeedback` ....